lib/components/narrative/metro/metro-error-renderer.tsx

```diff
@@ -57,6 +65,9 @@ const ErrorRenderer = ({ errors }: { errors: Error }): JSX.Element => {
           return null
         }
 
+        // Don't show errors that have been muted in the config
```

Comment:
Maybe move the two `return null` to the top of the mapping function.

Reply:
Good catch thanks!